File: java/Samples/Biometrics/CS/ServerSampleCS/Connections/ITemplateLoader.cs
using System;
using Neurotec.Biometrics;

namespace Neurotec.Samples.Connections
{
	public interface ITemplateLoader : IDisposable
	{
		void BeginLoad();
		void EndLoad();
		bool LoadNext(out NSubject[] subjects, int n);
		int TemplateCount { get; }
	}
}

File: java/Samples/Biometrics/VB/ServerSampleVB/Connections/ITemplateLoader.vb
Imports System
Imports Neurotec.Biometrics

Public Interface ITemplateLoader
	Inherits IDisposable
	Sub BeginLoad()
	Sub EndLoad()
	Function LoadNext(ByRef subjects() As NSubject, ByVal n As Integer) As Boolean
	ReadOnly Property TemplateCount() As Integer
End Interface
